### DISCUSSION TITLE
Firefox 108 + TMP 1.0.0-pre.11 Address bar not working

### DISCUSSION THREAD

**Discussion by sillysailor** [onemen/TabMixPlus#198] [Q&A] [answered]
I have downloaded TMP and it works almost flawlessly, except when I try to enter something into the address bar, nothing happens. I have downloaded the test build that was supposed to fix the problem to no avail.

**Comment by onemen** [comment, marked as answer]
I've just tested the latest test build [1.0.0-pre.12.20221215.1.xpi](https://tabmixplus.org/dev-builds/test%20builds/1.0.0-pre.12.20221215.1.xpi) on Firefox 108 and everything work including the address bar

**Reply by sillysailor** [reply]
Thank you so much for your work, onemen!!
